test/integration.test.js

```diff
@@ -212,24 +213,23 @@ describe('with access key', function () {
             aliceAccountName);
         await nearjs.waitForTransactionResult(createAccountResponse);
         await keyStore.setKey(newAccountId, newAccountKeyPair);
-    });
-
-    afterEach(async () => {
-        console.log = oldLog;
-    });
 
-    test('make function calls using access key', async () => {
-        // Adding access key
-        const keyForAccessKey = KeyPair.fromRandomSeed();
         const addAccessKeyResponse = await account.addAccessKey(
```

Comment:
Does this need to happen for literally every test in the file? 

Generally similar tests should be grouped using `describe`, then you can have common init code for tests in the group isolated from other tests. This can be nested multiple times. That's whole idea of RSpec-like test frameworks.

Like you nest stuff something like this:
- with node connection
    - with deployed contract
         - with access key
              - make function calls
              - view account details
              - add second key
              - remove access key

See e.g. https://github.com/mochajs/mocha/blob/master/test/unit/runnable.spec.js
Or array example here https://rspec.info/documentation/3.8/rspec-core/#A_Word_on_Scope

Reply:
I prefer the way they are grouped now, because I am not testing local node connection but access keys. The top level thing should describe the broad category you are testing, not setup steps. I think we should not block PRs over a stylistic disagreements over how to organize things right before release IMO.

Reply:
right now access keys setup happens for literally every test in the file. Is this what you intended? This is not just stylistic, it's also test performance issue.

Reply:
never mind, I misread the diff and thought it got move to top-level, keeping within `describe('with access key'` is fine